migration-cluster-day2/charts/migration-cluster-day2/templates/lvm-storage-operator.yaml

```diff
@@ -1,43 +1,3 @@
----
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: {{ .Values.lvmsOperator.subscription.namespace }}
-  # prevent the namespace deletion because in some sceneraious the namespace
-  # is not removed and stays in terminating state.
-  annotations:
-    argocd.argoproj.io/sync-wave: "-1"
-  labels:
-    argocd.argoproj.io/managed-by: openshift-gitops
----
-apiVersion: operators.coreos.com/v1
-kind: OperatorGroup
-metadata:
-  name: lvm-storage-operator-group
-  namespace: {{ .Values.lvmsOperator.subscription.namespace }}
-  labels: {{ include "migration-cluster-day2.labels" . | nindent 4 }}
-  annotations:
-    argocd.argoproj.io/sync-wave: "1"
-spec:
-  targetNamespaces:
-    - {{ .Values.lvmsOperator.subscription.namespace }}
----
-apiVersion: operators.coreos.com/v1alpha1
-kind: Subscription
-metadata:
-  name: lvm-storage-operator 
-  namespace: {{ .Values.lvmsOperator.subscription.namespace }}
-  labels: {{ include "migration-cluster-day2.labels" . | nindent 4 }}
-  annotations:
-    argocd.argoproj.io/sync-wave: "1"
-spec:
-  channel: {{ .Values.lvmsOperator.subscription.channel }}
-  installPlanApproval: Automatic
-  name: lvms-operator
-  source: redhat-operators 
-  sourceNamespace: openshift-marketplace
----
-
 allowVolumeExpansion: true
```

Comment:
So do I understand corectly that you now rely on assisted installer to configure LVMCluster CR, but the only think you can't configure is the storage class, so you do it using gitops?

Reply:
Cool! Can you please investigate if you can use this feature:
https://github.com/openshift/assisted-installer/pull/940

To setup storage class?

Reply:
wow this is a super cool feature! I will investigate it, thank you!